sys/fs/fuse/fuse_internal.c

```diff
@@ -1115,7 +1114,7 @@ fuse_internal_send_init(struct fuse_data *data, struct thread *td)
 	 * FUSE_MAX_PAGES: not yet implemented
 	 */
 	fiii->flags = FUSE_ASYNC_READ | FUSE_POSIX_LOCKS | FUSE_EXPORT_SUPPORT
-		| FUSE_BIG_WRITES | FUSE_WRITEBACK_CACHE
+		| FUSE_BIG_WRITES | FUSE_HAS_IOCTL_DIR | FUSE_WRITEBACK_CACHE
```

Comment:
I think we also need to check for FUSE_HAS_IOCTL_DIR during fuse_internal_init_callback.  If not present, then we should return ENOTTY whenever a user tries to use an ioctl on a directory.